File: doc/src/scripts/SurfacePrograde.py
import krpc
conn = krpc.connect(name='Surface prograde')
vessel = conn.space_center.active_vessel
ap = vessel.auto_pilot

ap.reference_frame = vessel.surface_velocity_reference_frame
ap.target_direction = (0,1,0)
ap.engage()
ap.wait()
ap.disengage()

File: doc/src/scripts/SurfacePrograde.cs
using System;
using KRPC.Client;
using KRPC.Client.Services.SpaceCenter;

class SurfacePrograde
{
    public static void Main ()
    {
        var connection = new Connection (name : "Surface prograde");
        var vessel = connection.SpaceCenter ().ActiveVessel;
        var ap = vessel.AutoPilot;

        ap.ReferenceFrame = vessel.SurfaceVelocityReferenceFrame;
        ap.TargetDirection = new Tuple<double,double,double> (0, 1, 0);
        ap.Engage ();
        ap.Wait ();
        ap.Disengage ();
    }
}
